code/modules/mob/living/simple_animal/friendly/sloth.dm

```diff
@@ -0,0 +1,41 @@
+/mob/living/simple_animal/sloth
```

Comment:
The only thing in /pet seemed to be code for adding pet collars, and I don't want to make new overlays for sloth collars.